cdist/conf/type/__user_groups/gencode-remote

```diff
@@ -20,6 +20,17 @@
 
 user="$(cat "$__object/parameter/user" 2>/dev/null || echo "$__object_id")"
 state_should="$(cat "$__object/parameter/state")"
+oldusermod="$(cat "$__object/explorer/oldusermod")"
+
+if [ "$oldusermod" = "true" ]; then
+        old=yes
+        addparam="-A"
+        delparam="-R"
+else
+        old=no
```

Comment:
Minor improvement possible: if we have $oldusermod, we do not need to setup old=, but can use $oldusermod below again :-)